confutils.nim

```diff
@@ -394,8 +394,14 @@ func getNextArgIdx(cmd: CmdInfo, consumedArgIdx: int): int =
   -1
 
 proc noMoreArgsError(cmd: CmdInfo): string =
-  result = if cmd.isSubCommand: "The command '$1'" % [cmd.name]
-           else: appInvocation()
+  result =
+    if cmd.isSubCommand:
+      try:
+        "The command '$1'" % [cmd.name]
+      except ValueError as err:
```

Comment:
With this exception caught, are any other exceptions possible from `noMoreArgsError`? If not, then its signature can be annotated not to throw exceptions.

Reply:
1c4679b18e71875bdcd51d96d77442701e4df6fe